API/Entities/AppUser.cs

```diff
@@ -1,13 +1,25 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 namespace API.Entities
 {
     public class AppUser
     {
         public int Id { get; set; }
 
+        [Required]
+        [MaxLength(50)]
         public string Username { get; set; }
 
+        [Required]
         public byte[] PasswordHash { get; set; }
         
+        [Required]
         public byte[] PasswordSalt { get; set; }
+
+        public IEnumerable<Saldo> Saldos { get; set; }
```

Comment:
Ogolnie to wymyslasz slowa. Nie ma po angielsku czegos takiego jak Saldo, musisz poprawic